.github/workflows/checks.yml

```diff
@@ -28,3 +28,29 @@ jobs:
       - run: yarn
       - run: yarn run prettify:diff
       - run: yarn run lint
+  mobile-tests:
+    name: Mobile Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+          check-latest: true
+      - name: Install Yarn dependencies
+        run: yarn
+      - name: Build Yarn packages
+        run: yarn build --ignore docs
+      - name: Verify locales if using l10n/* branch
+        run: |
+          if [[ "${GITHUB_HEAD_REF#refs/heads/}" != l10n/* ]]; then
+            # We want to check translations only when they are complete
+            echo "Skipping translations check on non l10n branches"
+            exit 0
+          fi
+          cd packages/mobile
+          yarn --cwd packages/mobile test:verify-locales
+      - name: Run mobile tests
```

Comment:
Can we upload the Istanbul coverage report directory as an artifact? it's location should be `packages/mobile/coverage`

```
- uses: actions/upload-artifact@v2
  with:
    name: coverage-report
    path: packages/mobile/coverage
```

Reply:
added